src/cascadia/TerminalApp/AppLogic.h

```diff
@@ -38,6 +38,7 @@ namespace winrt::TerminalApp::implementation
         winrt::Windows::UI::Xaml::ElementTheme GetRequestedTheme();
         LaunchMode GetLaunchMode();
         bool GetShowTabsInTitlebar();
+        float CalcSnappedDimension(const bool widthOrHeight, const float dimension) const;

```

Comment:
I'd prefer the name "isWidth" or "isHeight" depending on which one true is equal to as it's clearer what you're actually talking about here without further commenting when it's phrased that way.

An enum is probably overkill, I imagine.

Reply:
That's #3181

Reply:
That's fine, it can be deferred to there. Thanks.